src/components/Login/Login.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useState } from 'react';
+import classes from './Login.module.scss';
+import Input from '../core/Input/Input';
+import Button from '../core/Button/Button';
+
+interface CredentialsProps {
+  username: string;
+  password: string;
+}
+
+const Login = () => {
+  const [credentials, setCredentials] = useState<CredentialsProps>({
+    username: '',
+    password: ''
+  });
+
+  const handleLogin = () => {
+    //login logic
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setCredentials((prev) => ({ ...prev, [event.target.name]: event.target.value }));
+  };
+
+  return (
+    <div className={classes['c-login']}>
+      <div className={classes['c-login__container']}>
+        <div className={classes['-login__container__span--container']}>
```

Comment:
Here you are missing `c` part, also I would like for you to name this differently since this is not really readable. For example it can be `c-login__input-container`. Also there is no need to chain it with container